.github/workflows/runtests.yml

```diff
@@ -114,53 +114,38 @@ jobs:
       - run: bin/doctest --force-colors
       - run: examples/all.py -q
 
-  # -------------------- Test split 1/2 latest Python -------------- #
+  # ------------------------- Test latest Python ------------------- #
 
-  tests1-latest:
-    needs: code-quality
-
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
-        with:
-          python-version: '3.10'
-      - run: python -m pip install --upgrade pip
-      - run: pip install mpmath
-      - run: bin/test --force-colors --split=1/2
-
-  # -------------------- Test split 2/2 latest Python -------------- #
-
-  tests2-latest:
+  tests-latest:
     needs: code-quality
 
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        group: [1, 2, 3, 4]
```

Comment:
I think you want to set `fail-fast: false` here so that all jobs complete even if one fails. Possibly also `continue-on-error`. I can't remember which of these does what.

Reply:
Addressed in https://github.com/sympy/sympy/pull/24291/commits/dae44c0568ecb0d379333feaeb2f7a0a95037482